tests/acceptance/bootstrap/GraphContext.php

```diff
@@ -2819,6 +2819,55 @@ public function userListsTheActivitiesForResourceOfSpaceUsingTheGraphAPI(string
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When user :user tries to list the activities of folder :folder from space :spaceName using share mount-point id using the Graph API
```

Comment:
space should be always `Shares` for this step
```suggestion
	 * @When user :user tries to list the activities of folder :folder with share mount-point-id using the Graph API
```